packages/frameworks/src/frameworks.ts

```diff
@@ -0,0 +1,1605 @@
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        value: 'bundle install',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `jekyll build`',
+      },
+      devCommand: {
+        value: 'bundle exec jekyll serve --watch --port $PORT',
+      },
+      outputDirectory: {
+        placeholder: '`_site` or `destination` from `_config.yml`',
+      },
+    },
+    devCommand: 'bundle exec jekyll serve --watch --port $PORT',
+    buildCommand: 'jekyll build',
+    getOutputDirName: async (dirPrefix: string): Promise<string> => {
+      type JekyllConfig = { destination?: string };
+      const config = await readConfigFile<JekyllConfig>(
+        join(dirPrefix, '_config.yml')
+      );
+      return (config && config.destination) || '_site';
+    },
+  },
+  {
+    name: 'Brunch',
+    slug: 'brunch',
+    demo: 'https://brunch.now-examples.now.sh',
+    logo:
+      'https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/brunch.svg',
+    tagline:
+      'Brunch is a fast and simple webapp build tool with seamless incremental compilation for rapid development.',
+    description: 'A Brunch app, created with the Brunch CLI.',
+    website: 'https://brunch.io/',
+    detectors: {
+      every: [
+        {
+          path: 'brunch-config.js',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `brunch build --production`',
+      },
+      devCommand: {
+        value: 'brunch watch --server --port $PORT',
+      },
+      outputDirectory: {
+        value: 'public',
+      },
+    },
+    devCommand: 'brunch watch --server --port $PORT',
+    buildCommand: 'brunch build --production',
+    getOutputDirName: async () => 'public',
+  },
+  {
+    name: 'Middleman',
+    slug: 'middleman',
+    demo: 'https://middleman.now-examples.now.sh',
+    logo:
+      'https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/middleman.svg',
+    tagline:
+      'Middleman is a static site generator that uses all the shortcuts and tools in modern web development.',
+    description: 'A Middleman app, created with the Middleman CLI.',
+    website: 'https://middlemanapp.com/',
+    detectors: {
+      every: [
+        {
+          path: 'config.rb',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        value: 'bundle install',
+      },
+      buildCommand: {
+        value: '`npm run build` or `bundle exec middleman build`',
+      },
+      devCommand: {
+        value: 'bundle exec middleman server -p $PORT',
+      },
+      outputDirectory: {
+        value: 'build',
+      },
+    },
+    devCommand: 'bundle exec middleman server -p $PORT',
+    buildCommand: 'bundle exec middleman build',
+    getOutputDirName: async () => 'build',
+  },
+  {
+    name: 'RedwoodJS',
```

Comment:
Any reason why you changed the order of RedwoodJS and Docusaurus 1?

Reply:
Accidental, I'll fix.